.gitignore

```diff
@@ -59,4 +59,6 @@ venv.bak/
 .terraform.lock.hcl
 .terraform/
 terraform.tfstate
-terraform.tfstate.backup
\ No newline at end of file
+terraform.tfstate.backup
+detectors/apiScripts/.vscode/launch.json
+detectors/apiScripts/.vscode/settings.json
```

Comment:
may be best to replace these two lines with a more general entry like
```
# VS Code files for those working on multiple tools
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
*.code-workspace
```
then it will ignore those vscode files anywhere since others probably don't want to add them either.